### ISSUE TITLE
Folder Monitor progress bar?

### ISSUE THREAD

**Issue by Andy-ABTec** [TV-Rename/tvrename#217]
Just taking a wander in **_Tools>>Folder Monitor_** and found this...
![2017-12-28_121626](https://user-images.githubusercontent.com/5175695/34410695-f0d8d166-ebc9-11e7-932c-fefae2ceed7a.png)
It appears when "Check >>" is clicked.
The check is fine but the progress bar does nothing, maybe it should show % complete (difficult I should think, as you don't know how much you're scanning until you've scanned it), or a "Cylon" style scan to indicate activity or not be there at all?

**Comment by JoeBiellik** [comment]
You are correct, it's slower to count the number of folders, for percentage, before processing them. This should be changed to a marquee (Cylon) style, or simply changed so the list of folders populates as they process time rather than all at the end.

**Comment by Andy-ABTec** [comment]
I'm no coder so please shoot me down!
Both would be really cool, a sweeping "Cylon" style eye somewhere on the pane AND populate the list as stuff is found!

**Comment by MarkSummerville** [comment]
Not sure what the Cylon style is, but I agree that something should be done to improve this form

**Comment by JoeBiellik** [comment]
In v3, I've removed this little popup window entirely and am showing the progress bar directly in the form.
